src/data_structures.rs

```diff
@@ -66,7 +91,7 @@ pub trait PCPreparedCommitment<UNPREPARED: PCCommitment>: Clone {
 
 /// Defines the minimal interface of commitment randomness for any polynomial
 /// commitment scheme.
-pub trait PCRandomness: Clone {
+pub trait PCRandomness: Clone + Debug + Eq + PartialEq + CanonicalSerialize + CanonicalDeserialize {
```

Comment:
Why do we need implementations for ``Eq`` and ``ParitalEq`` here?

Reply:
One guess: for the serialize-deserialize tests?

Reply:
You answered yourself :D